docs/install.md

```diff
@@ -0,0 +1,132 @@
+    `default` key, and additional keys allowing modules to use their own
+    database to avoid stepping on each other's toes. This is especially useful
+    for bespoke modules created for the needs of a specific site, which can thus
+    use their own databases, possibly located on other MongoDB clusters.
+    For example, with the following settings:
+
+```php
+// In sites/default/settings.local.php.
+$settings['mongodb'] = [
+  'clients' => [
+    // Client alias => connection constructor parameters.
+    'default' => [
+      'uri' => 'mongodb://localhost:27017',
+      'uriOptions' => [],
+      'driverOptions' => [],
+    ],
+  ],
+  'databases' => [
+    // Database alias => [ client_alias, database_name ]
+    'default' => ['default', 'drupal'],
+    'keyvalue' => ['default', 'keyvalue'],
+    'logger' => ['default', 'logger'],
+  ],
+];
+```
+  * With these settings:
+    * the `default` database alias will handle collections in the `drupal`
+      database on the `default` MongoDB server installed in earlier steps
+    * the `keyvalue` database alias will store key-value collections on the
+      same `default` MongoDB server, but in a separate `keyvalue` database.
+    * the `logger` database alias will store logger collections on the same
+      `default` MongoDB server, but in a separate `logger` database.
+* To use the MongoDB Key-Value (Expirable) storage:
+  * ensure there is a `keyvalue` database alias in `settings.local.php`, like
+    in the previous lines.
+  * declare MongoDB as the default keyvalue storage implementation by editing
+    the existing declarations in the `sites/default/services.yml` file:
+
+```yaml
+# In sites/default/services.yml.
+factory.keyvalue:
+  default: keyvalue.mongodb
+factory.keyvalue.expirable:
+  keyvalue_expirable_default: keyvalue.expirable.mongodb
+```
+
+  * enable the `mongodb_storage` module, e.g. using `drush en mongodb_storage`.
+  * import the existing Key-Value contents from the database, using the Drush
+    `mongodb_storage-import-keyvalue` command: `drush most-ikv`. It will output
+    the list of imported keys, for your information, like:
+
+```yaml
+key_value
+  config.entity.key_store.action
+    uuid:054e62b3-1c40-4f22-aa17-c092bd796ee8
+    uuid:0cfd15f5-c01a-4912-991c-ad10e934f86e
+(...lots of line, then...)
+key_value_expire
+  update_available_releases
+    drupal
+```
+
+  * rebuild the container to take these changes into account using `drush cr`.
+
+Once the module is installed and enabled, you can check its requirements on
+`/admin/reports/status`:
+
+![MongoDB on status page](images/mongodb-requirements.png)
+
+
+##Composer Requirements
+
+* Commands below are for those who are using composer already in your site to
+  manage module dependencies. To know more about composer [here][composer].
+
+[composer]: https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies
+
+* At the root of your site
```

Comment:
Indentation in this section is not correct in mkdocs, some code blocks appear inline, and some levels are not indented. AIUI mkdocs needs more than two spaces in some case: be sure to have a version looking as it should locally, which is not the case currently, at least on my machine.